### PR TITLE
Revup cherry-pick attempts to fetch

### PR DESCRIPTION
**Pull request by aaron-skydio**
Will only attempt to fetch if the branch isn't found locally, and will
still fail if it isn't found after fetching

Topic: cherry-pick-fetch
Reviewers: jerry,brian-k

### CODE REVIEW

**Comment by aaron-skydio** [comment]
Reviews in this chain:
└**https://github.com/Skydio/revup/pull/117** **Revup cherry-pick attempts to fetch**

**Comment by aaron-skydio** [comment]
| # | head | base | diff | date | summary |
| - | - | - | - | - | - |
| 0 | [286b8654](/skydio/revup/commit/286b8654fe3b1c4f711395a20436ee758547302d) | [917438e8](/skydio/revup/commit/917438e84f605f44a998dc305eeab63479cab9f7) | [diff](/skydio/revup/compare/917438e84f605f44a998dc305eeab63479cab9f7..286b8654fe3b1c4f711395a20436ee758547302d) | Jun 1 16:40 PM |  1 file changed, 19 insertions(+), 10 deletions(-) |

File: revup/cherry_pick.py [hunk 1]
```diff
@@ -13,17 +12,27 @@ async def main(args: argparse.Namespace, git_ctx: git.Git) -> int:
...
     remote_branch_to_pick = git_ctx.ensure_branch_prefix(branch_to_pick)
-    branch_exists, remote_branch_exists = await asyncio.gather(
-        git_ctx.is_branch_or_commit(branch_to_pick),
-        git_ctx.is_branch_or_commit(remote_branch_to_pick),
-    )
-    if remote_branch_exists and not branch_exists:
+    branch_exists = await git_ctx.is_branch_or_commit(branch_to_pick)
+
+    if not branch_exists:
         logging.warning(
```
Comment (jerry-skydio) [inline revup/cherry_pick.py:18]: these can probably just be info instead of warning now

**Review by jerry-skydio — APPROVED** [review]

File: revup/cherry_pick.py (hunk 1 above)
Comment (aaron-skydio) [reply revup/cherry_pick.py]: Done

Comment (aaron-skydio) [reply revup/cherry_pick.py]: Will let you hit the merge button

**Comment by aaron-skydio** [comment]
Had to add a `--force`, which is required if the branch has already been fetched - I think this is probably fine?

**Pull request merged**